app/views/stock_status/stock_status_by_project_code.html.erb

```diff
@@ -0,0 +1,101 @@
+<% @warehouses = Warehouse.all %>
+<div class="row wrapper border-bottom white-bg page-heading">
+  <div class="col-sm-4">
+    <h2>Stock Status</h2>
+  </div>
+  <div class="col-sm-8">
+    <div class="title-action">
+    </div>
+  </div>
+</div>
+<div class="ibox float-e-margins">
+  <div class="ibox-content">
+      <%= form_tag( "/stock_status/stock_status_by_project_code" ,{ class: 'form-inline', style: 'display: inline-block;', role: 'form', method: 'GET'}) do %>
+
+      	<div class="form-group">
+          <%= label_tag :hub, "Hub", { :class => ' control-label'} %>
+
+          <div class="">
+             <%= collection_select :hub_id,:id,Hub.order(:name), :id, :name, {:selected => Warehouse.find_by(id: @location&.warehouse_id)&.hub_id,  :prompt => true},  {  :id => 'hub_select',:name => 'hub' ,:class => 'form-control' }%>
+          </div>
+        </div>
+        <div class="form-group">
+          <%= label_tag :program, "warehouse", { :class => ' control-label', :required => true } %>
+
+          <div class="">
+             <%= collection_select :warehouse_id,:id,@warehouses.select{|w| w.hub_id == "#{:hub_select}"}, :id, :name, { :prompt => true}, {:id => 'warehouse_select',:name => 'warehouse' ,:class => 'form-control' }%>
+          </div>
+        </div>
+        
+        <div class="form-group">
+          <label for=""></label>
+          <div>
+              <input type="submit" name="filter" value="Go" class="btn btn-sm btn-primary" />
```

Comment:
When I try to submit this filter, the form doesn't keep my hub and warehouse choices.